src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -166,18 +182,32 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 	res.render("jira-configuration-new.hbs", {
 		host: jiraHost,
 		gheServers: groupedGheServers,
-		ghCloud: { successfulConnections, failedConnections },
-		hasCloudAndEnterpriseServers: !!((successfulConnections.length || failedConnections.length) && gheServers.length),
-		hasCloudServers: !!(successfulConnections.length || failedConnections.length),
+		ghCloud: { successfulCloudConnections, failedCloudConnections },
+		hasCloudAndEnterpriseServers: !!((successfulCloudConnections.length || failedCloudConnections.length) && gheServers.length),
+		hasCloudServers: !!(successfulCloudConnections.length || failedCloudConnections.length),
 		hasConnections: !!(installations.total || gheServers?.length),
 		APP_URL: process.env.APP_URL,
 		csrfToken: req.csrfToken(),
 		nonce
 	});
+
+	const successfulServerConnections = gheServersWithConnections
+		.reduce((acc, obj) => acc + obj.successfulConnections?.length, 0);
+	const failedServerConnections = gheServersWithConnections
+		.reduce((acc, obj) => acc + obj.failedConnection?.length, 0);
+	const allSuccessfulConnections = [...successfulCloudConnections, ...gheServersWithConnections];
+
 	sendAnalytics(AnalyticsEventTypes.ScreenEvent, {
 		name: AnalyticsScreenEventsEnum.GitHubConfigScreenEventName,
 		jiraHost,
-		connectedOrgCount: installations.total + gheServers.length
+		connectedOrgCountCloudCount: successfulCloudConnections.length,
+		connectedOrgCountServerCount: successfulServerConnections,
+		totalOrgCount: successfulCloudConnections.length + successfulServerConnections,
+		failedCloudBackfillCount: countStatus(failedCloudConnections, "FAILED"),
+		failedServerBackfillCount: countStatus(failedServerConnections, "FAILED"),
+		successfulCloudBackfillCount: countStatus(successfulCloudConnections, "FINISHED"),
+		successfulServerBackfillCount: countStatus(successfulServerConnections, "FINISHED"),
+		numberOfSkippedRepos: countNumberSkippedRepos(allSuccessfulConnections)
```

Comment:
Metrics added here:
- number of successful and failed connections per product (cloud | server)
- number of connected orgs per product
- total number of connected orgs (easy way to create a chart with total number but may not be needed as we may be able to add these numbers in redash... thoughts?)
- the number of repos skipped: will give us insight how frequently we're skipping backfilling repos for each customer